### PR TITLE
Implement/Replace SES

### PR DESCRIPTION
**Pull request by kaimanasse15**
replace/implement this function in the backend to make it connect to the AWS SES.

### CODE REVIEW

**Comment by brinkbrink** [comment]
Hey @kaimanasse15 it looks like we have some tests failing here. Please take a look and reach out to @tinpham5614, @Exochos, @Robel-003, or @bcko. We should also have ample time tonight for troubleshooting.

**Review by Exochos — COMMENTED** [review]
This code looks like it removes all of the user auth and login stuff, we still want all of that. We are  just using SES for the one particular use case.

**Comment by kaimanasse15** [comment]
I will fix it. Thanks, Jeremy. I thought that I need to replace everything.

File: src/auth/services/auth.service.ts
Comment (kaimanasse15) [inline src/auth/services/auth.service.ts:1]: There is a part call "  throw new HttpException('Failed to send email', 500);". Would anyone help me to fix it? Thank you

**Comment by brinkbrink** [comment]
Hey @kaimanasse15 is this still something you want help on? Should we close this? @tinpham5614

**Comment by tinpham5614** [comment]
Please close it for now, I need more time to work on this.
Thanks, Brink

**Comment by brinkbrink** [comment]
Okay, sounds good @tinpham5614

**Pull request closed without merging**